evalml/automl/automl_search.py

```diff
@@ -711,7 +711,7 @@ def __init__(
         self.search_parameters = search_parameters or {}
         # Fitting takes a long time if the data is too wide or long.
         if is_time_series(problem_type) and (
-            self.X_train.shape[1] >= 10 or self.X_train.shape[0] >= 10000
+            self.X_train.shape[1] >= 7 or self.X_train.shape[0] >= 1000
```

Comment:
can we bake this in a constant somewhere and use this constant in the test as well?